### PR TITLE
Include sonr logo image on background

### PR DESCRIPTION
**Pull request by hvhallmann**
Simulator Screen Shot - iPhone 12 Pro - 2022-07-18 at 10 47 16

Simulator Screen Shot - iPhone 13 - 2022-07-19 at 14 27 44

### CODE REVIEW

**Comment by hvhallmann** [comment]
It seems necessary to change the image color,
but let me know if this a desired path.
It was the only way I found to include this image, maybe I can try with svg

**Comment by hvhallmann** [comment]
yes, I need to download the light one.
Ok, when I have some time I'll update this one

**Comment by hvhallmann** [comment]
ok, now we have the light background also

File: src/components/ContainerLight.tsx [hunk 1]
```diff
@@ -1,14 +1,29 @@
 import React from "react";
-import { StyleSheet, View } from "react-native";
+import { ImageBackground, StyleSheet, View } from "react-native";
+
+const logo = require("../icons/logo_background_white.png");
```
Comment (tgfassina) [inline src/components/ContainerLight.tsx:4]: Works alright. But why does it have to be a png?

Comment (tgfassina) [reply src/components/ContainerLight.tsx:4]: @hvhallmann

Comment (hvhallmann) [reply src/components/ContainerLight.tsx:4]: ow sorry, I didnt see this question,
but I could not make it work with svg.

**Pull request merged**